### ISSUE TITLE
Feature Request, in option add branch name in commit message

### ISSUE THREAD

**Issue by w0nderw0man** [ForkIssues/TrackerWin#527]
I <3 your tool,
Feature Request : can we have a control to add the current branch name in commit message,

Thanks

**Comment by DanPristupov** [comment]
Yes, it would be nice to introduce some variables which could be used in commit templates.

* $current branch
* $repositoryname
* $date

**Comment by sm-g** [comment]
> $current branch

note: with GitFlow branches have names like `feature/XX-1234`, and desirable commit template is `XX-1234:`
